components/brave_rewards/resources/android_page/reducers/rewards_reducer.ts

```diff
@@ -124,11 +124,12 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
       }
 
       state.adsData.adsEnabled = action.payload.adsData.adsEnabled
+      state.adsData.shouldAllowAdConversionTracking = action.payload.adsData.shouldAllowAdConversionTracking
       state.adsData.adsPerHour = action.payload.adsData.adsPerHour
       state.adsData.adsUIEnabled = action.payload.adsData.adsUIEnabled
       state.adsData.adsIsSupported = action.payload.adsData.adsIsSupported
       break
-    }
+    } 
```

Comment:
Nit: looks like an extra space sneaked in.

Reply:
fixed